src/commands/dev/mod.rs

```diff
@@ -1,19 +1,29 @@
 mod server_config;
+mod socket;
 use server_config::ServerConfig;
 mod headers;
 use headers::{prepend_request_headers_prefix, strip_response_headers_prefix};
 
+extern crate openssl;
+extern crate tokio;
+extern crate url;
+extern crate ws;
+
+use std::thread;
+
 use chrono::prelude::*;
 
 use hyper::client::{HttpConnector, ResponseFuture};
 use hyper::header::{HeaderName, HeaderValue};
 
 use hyper::http::uri::InvalidUri;
 use hyper::service::{make_service_fn, service_fn};
-use hyper::{Body, Client, Request, Response, Server, Uri};
+use hyper::{Body, Client as HyperClient, Request, Response, Server, Uri};
 
 use hyper_tls::HttpsConnector;
 
+use tokio::runtime::Runtime;
```

Comment:
nit: maybe `as TokioRuntime`? not that we have a boatload of runtimes running around, but is a pretty vague constructor name.

Reply:
done